Shorts/Stacks/stack_ops/test-unit/test.cpp

```diff
@@ -0,0 +1,122 @@
+#include "../src/stack_ops.h"
+#include <stack>
+#define CATCH_CONFIG_MAIN
+#include "catch.hpp"
+
+/*
+	To check output (At the stack_ops directory):
```

Comment:
For this problem, it seems like Edugator has 5 tests while this has 7. Is there a reason why?